features/manage-position/Create.tsx

```diff
@@ -43,6 +45,7 @@ const MinLink = styled.div`
 const { formatUnits: fromWei, parseBytes32String: hexToUtf8 } = utils;
 
 const Create = () => {
+  const { network } = Connection.useContainer();
   const { contract: emp } = EmpContract.useContainer();
   const { empState } = EmpState.useContainer();
   const {
```

Comment:
I am unable to comment below but should this PR not also change the `setTokensToMax` function to accommodate the UMIP-14 changes? the `max` amount of tokens to mint for a given amount of collateral should change accordingly, right?

Reply:
oh great point. I'll fix